README.md

```diff
@@ -1,61 +1,57 @@
 # SyncPromise
 
-A fast, small, _safe_ promise implementation with synchronous promise resolution
-and an API which resembles ECMAScript promises.
+A fast, small, _safe_ promise implementation with synchronous promise
+resolution and an API which resembles `ECMAScript` promises.
```

Comment:
I'm not sure if code tags are appropriate in a README for things that aren't code.